catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/reader/GetRecordsMessageBodyReader.java

```diff
@@ -59,6 +62,10 @@
 public class GetRecordsMessageBodyReader implements MessageBodyReader<CswRecordCollection> {
     private static final Logger LOGGER = LoggerFactory.getLogger(GetRecordsMessageBodyReader.class);
 
+    public static final String BYTES_SKIPPED = "bytes-skipped";
+
```

Comment:
Thanks! I couldn't find it in any of the other HttpHeader classes we use for some reason. :+1: